pkg/kernel/kernel_config-4.19.x-aarch64

```diff
@@ -635,7 +635,8 @@ CONFIG_KVM=y
 CONFIG_KVM_ARM_HOST=y
 CONFIG_KVM_ARM_PMU=y
 CONFIG_KVM_INDIRECT_VECTORS=y
-# CONFIG_VHOST_NET is not set
+CONFIG_VHOST_NET=m
+CONFIG_VHOST=m
```

Comment:
Any reason for these being modules for arm and in the kernel for intel?

Reply:
No reason -- this is just how it came out of make menuconfig on ARM. My suspicion is that making it "y" will pull in a lot of other pieces that are currently "m" on ARM. But it is a good spot -- I'll try to build it on ARM with replacing m with y and see what happens before I merge.

Reply:
Note that the introduction of the sysctl for soft_lockup seems to have hosed things on the hikey. zedrouter hangs when issuing that sysctl for some reason. Just a heads up if you are going to test what you build.

Reply:
So first -- yes turning it into 'y' pulls in WAY too much stuff that I don't have to deal with (the HiKey kernel is weird because it refused to boot with too many modules linked statically into the kernel).

On the soft_lockup -- see my reply on the other PR -- I did what I did for a reason there ;-)